src/Components/PresentationalComponents/InsightsSystemRule/InsightsSystemRule.js

```diff
@@ -63,3 +63,24 @@ export const InsightsNoSystemRule = ({ cve }) => {
 InsightsNoSystemRule.propTypes = {
     cve: PropType.string
 };
+
+export const InsightsNotVulnerable = ({ cve, mitigationReason }) => {
+    return (
+        <TextContent>
+            <Text component={TextVariants.h3}>
+                <FormattedMessage {...messages.notVulnerableBoxTitle} />
```

Comment:
Somewhere you use `intl.formatMessage` (https://github.com/RedHatInsights/vulnerability-ui/pull/1784/files#diff-fb097f9e1634d4ca593c2fd47a9cb539b82fed0a3593ada639f4ed7d5175a9c8R39), somewhere FormattedMessage. It's not important, but I think staying consistent here won't be bad

Reply:
Yes, we are inconsistent on this and we have a ticket to make all these approaches consistent.